src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1,20 +1,3661 @@
+            context,
+            !landing_gear.is_down_and_locked() && !self.is_gear_lever_down,
+        );
+
+        if is_in_flight_gear_lever_up {
+            if self.should_disable_auto_brake_when_retracting.output() {
+                self.left_brake_green_output = 0.;
+                self.right_brake_green_output = 0.;
+                self.left_brake_yellow_output = 0.;
+                self.right_brake_yellow_output = 0.;
+            } else {
+                self.left_brake_green_output = 0.2; // Slight brake pressure to stop the spinning wheels
+                self.right_brake_green_output = 0.2; // Slight brake pressure to stop the spinning wheels
+            }
+        } else {
+            let green_used_for_brakes = self.normal_brakes_available
+                && self.anti_skid_activated
+                && !self.parking_brake_demand;
+
+            if green_used_for_brakes {
+                self.left_brake_green_output = self.left_brake_pilot_input;
+                self.right_brake_green_output = self.right_brake_pilot_input;
+                self.left_brake_yellow_output = 0.;
+                self.right_brake_yellow_output = 0.;
+            } else {
+                self.left_brake_green_output = 0.;
+                self.right_brake_green_output = 0.;
+                if !self.parking_brake_demand {
+                    // Normal braking but using alternate circuit
+                    self.left_brake_yellow_output = self.left_brake_pilot_input;
+                    self.right_brake_yellow_output = self.right_brake_pilot_input;
+                } else {
+                    // Else we just use parking brake
+                    self.left_brake_yellow_output = 1.;
+                    self.right_brake_yellow_output = 1.;
+
+                    // Special case: parking brake on but yellow can't provide enough brakes: green are allowed to brake for emergency
+                    if alternate_circuit.left_brake_pressure().get::<psi>()
+                        < Self::MIN_PRESSURE_PARK_BRAKE_EMERGENCY
+                        || alternate_circuit.right_brake_pressure().get::<psi>()
+                            < Self::MIN_PRESSURE_PARK_BRAKE_EMERGENCY
+                    {
+                        self.left_brake_green_output = self.left_brake_pilot_input;
+                        self.right_brake_green_output = self.right_brake_pilot_input;
+                    }
+                }
+            }
+        }
+
+        // Limiting final values
+        self.left_brake_yellow_output = self.left_brake_yellow_output.min(1.).max(0.);
+        self.right_brake_yellow_output = self.right_brake_yellow_output.min(1.).max(0.);
+        self.left_brake_green_output = self.left_brake_green_output.min(1.).max(0.);
+        self.right_brake_green_output = self.right_brake_green_output.min(1.).max(0.);
+    }
+
+    pub fn send_brake_demands(&mut self, norm: &mut BrakeCircuit, altn: &mut BrakeCircuit) {
```

Comment:
**thought**: Going back and forth with information like this is a bit "complex".
**suggestion**: As discussed, this should probably be changed to a similar `Controller` style we used elsewhere.

You know best if you'll soon again work on this type. If so you can leave this for that particular PR. If not, I'd suggest still changing it now.

Reply:
Agreed, I still have unknowns about how the brakes that are sim input dependant might be separated in different refresh rates updates. This might or might not impact the way we send those commands. We can change all of this in the PR where we send real brake commands to the sim. 
Hope it will be soon but I'll need SDK simvars gurus :D